src/components/organisms/navigation.tsx

```diff
@@ -34,15 +38,46 @@ const Navigation = ({ session }: INavigationProps) => {
         pos="sticky"
         top={8}
       >
-        <Link href="/">
-          <Icon
-            name="Logo"
-            color={`mode.${colorMode}.title`}
-            h={8}
-            w="auto"
-            cursor="pointer"
-          />
-        </Link>
+        <Flex
+          align="center"
+          backgroundColor={`mode.${colorMode}.card`}
+          rounded="sm"
+        >
+          <Link href="/">
+            <Icon
+              name="Logo"
+              color={`mode.${colorMode}.title`}
+              h={8}
+              w="auto"
+              cursor="pointer"
+            />
+          </Link>
+          <Breadcrumb ml={3} mt={2} addSeparator={true}>
```

Comment:
I see an issue with the separator being difficult to see. Does `Breadcrumb` have a color prop?
```suggestion
          <Breadcrumb ml={3} mt={2} addSeparator={true} color={`mode.${colorMode}.text`}>
```

Reply:
Yup! Just checked and it'll look like... 

<img width="506" alt="Screen Shot 2020-06-10 at 2 03 53 AM" src="https://user-images.githubusercontent.com/26869552/84212690-d0b12580-aabe-11ea-8385-fd7bece6ea4e.png">
<img width="503" alt="Screen Shot 2020-06-10 at 2 04 02 AM" src="https://user-images.githubusercontent.com/26869552/84212691-d149bc00-aabe-11ea-9c5b-334429892477.png">

Reply:
Also yeah sorry for the delay. Felt stuck last week and then suddenly yesterday randomly unstuck 😅 😂  the emotional rollercoaster of coding.